step.yml

```diff
@@ -105,30 +82,20 @@ inputs:
   - keystore_password: $BITRISEIO_ANDROID_KEYSTORE_PASSWORD
     opts:
       title: "Keystore password"
-      summary: ""
-      description: ""
       is_required: true
       is_sensitive: true
   - keystore_alias: $BITRISEIO_ANDROID_KEYSTORE_ALIAS
     opts:
-      title: "Keystore alias"
-      summary: ""
-      description: ""
+      title: "Key alias"
       is_required: true
       is_sensitive: true
   - private_key_password: $BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD
     opts:
-      title: "Private key password"
-      summary: ""
-      description: |
-        If key password equals to keystore password, you can leave it empty.
```

Comment:
Is the description no longer true?

Reply:
in the linked keystore doc: Password: Create and confirm a secure password for your key. This should be different from the password you chose for your keystore.

Reply:
Technically you can still use the same password, and the step behavior did nor change. So adding a "not recommended" not could be enough.